### DISCUSSION TITLE
Integration node ace add with bun package manager

### DISCUSSION THREAD

**Discussion by rust142** [adonisjs/core#4539] [General]
will there be a `node ace add` integration command with bun package manager?

![image](https://github.com/adonisjs/core/assets/56282560/1aaf4ebd-dd43-48b5-b8a9-59ee4898c65a)

**Comment by GabrielNBDS** [comment]
probably will be when bun supports adonis https://github.com/oven-sh/bun/issues/4822

**Comment by ShahriarKh** [comment]
Related: https://github.com/adonisjs/core/issues/4606

**Comment by thetutlage** [comment]
Added in https://github.com/adonisjs/core/pull/4607
